### ISSUE TITLE
Module doc return table issues

### ISSUE THREAD

**Issue by andy-maier** [ansible/ansible#25329]
##### ISSUE TYPE
 - Documentation Report

##### COMPONENT NAME
- Module documentation tool (plugin_formatter)
- docs/bin/plugin_formatter.py

##### ANSIBLE VERSION

```
$ PYTHONPATH=lib bin/ansible --version
ansible 2.4.0 (fix-module-doc-return-table 53c46df323) last updated 2017/06/03 16:10:19 (GMT +200)
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/maiera/virtualenvs/zhmcam27/lib/python2.7/site-packages/wheel/../ansible/modules']
  ansible python module location = /home/maiera/Projects/zhmcclient/repos/ansible/lib/ansible
  executable location = bin/ansible
  python version = 2.7.5 (default, Aug  2 2016, 04:20:16) [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
```

##### CONFIGURATION

defaults

##### OS / ENVIRONMENT

From: RHEL 7.3
Target: localhost

##### SUMMARY

I am developing an Ansible module and use the module documentation tool that Ansible also uses for its modules, namely `/docs/bin/plugin_formatter.py`.

For the module return table, that tool has two issues:

1. When specifying a list of strings in the `description` field of a return value, it shows them in python list syntax on the resulting web page,e.g. `['a', 'b', 'c'].

2. When specifying more than one line for the `sample` field, the result table gets damaged in the HTML output. I analyzed this and realized that the reason for that is that inthe RST file generated by plugin_formatter.py, the lines are not indented sufficiently.

##### STEPS TO REPRODUCE

For issue 1, specify a list of strings in the `description` field of a return value, and look at the result on the generated module web page.

For issue 2, specify a `sample` field with more than one line and look at the resuting RST file and HTML web page.

```yaml
N/A
```

##### EXPECTED RESULTS

For issue 1, I would expect a list of strings for the `description` field of return values to be supported just as they are supported for the `description` field of input parameters.

For issue 2, I would at least not expect the table to be damaged (i.e. the lines in RTS to be properly indented). More ideally, I would expect the `sample` string to be htmlified in addition (using the `html_ify` filter).

##### ACTUAL RESULTS

See summary

```
N/A
```

**Comment by andy-maier** [comment]
Fixed by PR #25330.

**Comment by acozine** [comment]
Not sure why this issue was not closed when https://github.com/ansible/ansible/pull/25330 was merged. Closing now. Please re-open if I've missed something.
